ecosystem/sep-0008.md

```diff
@@ -227,16 +251,13 @@ Name | Type | Description
 
 ### Best practices
 
+- Issuers using the `revised` status to make a transaction complaint is encouraged as it can take the burdens away from the wallets.
```

Comment:
```suggestion
- Issuers are encouraged to use the `revised` status to make a transaction complaint when possible, since Wallets may not know how to form a transaction that is compliant with the server's criteria.
```

Reply:
Thanks! Much better. https://github.com/stellar/stellar-protocol/pull/817/commits/0b464cafd42ad658e9b1281698abbedd61b4f380